### DISCUSSION TITLE
Local build is 4x slower than faiss release

### DISCUSSION THREAD

**Discussion by b8188495** [facebookresearch/faiss#3591] [General]
# Summary
I'm running the same benchmark on the same machine with a local build and the 1.8.0 release build. The local build is about 4x slower than the release. Can you share the build flags for a release so I can try it out with a local build?

My local build was created with the following commands on an AWS Graviton box (aarch64):
```
cmake -B build . -DFAISS_ENABLE_GPU=OFF -DFAISS_ENABLE_PYTHON=ON
make -C build -j faiss
make -C build -j swigfaiss
(cd build/faiss/python && python setup.py install)
(cd build/faiss/python && python setup.py build)
```

# Platform

OS: Ubuntu 20

Faiss version: dafdff110489db7587b169a0afee8470f220d295

Installed from: compiled locally

Faiss compilation options: 
cmake -B build . -DFAISS_ENABLE_GPU=OFF -DFAISS_ENABLE_PYTHON=ON

Running on:
- [X] CPU
- [ ] GPU

Interface: 
- [ ] C++
- [X] Python

# Reproduction instructions

**Comment by alexanderguzhva** [comment]
`-DCMAKE_BUILD_TYPE=Release` is missing in `cmake` command line.
Faiss builds the debug build by default
